web/docs/language/features.md

```diff
@@ -442,13 +444,13 @@ export const updateTaskIsDone = ({ id, isDone }, context) => {
 ```
 ```c title=main.wasp
 action updateTaskIsDone {
-  fn: import { updateTaskIsDone } from "@ext/actions.js",
+  fn: import { updateTaskIsDone } from "@server/actions.js",
   entities: [Task]
 }
 ```
 
 And here is how you might use it:
-```js {4,18} title=pages/Task.js
+```js {4,18} title=client/pages/Task.js
```

Comment:
I know the previous versions just used `pages`, but do we want to always preface with `src/` moving forward?